src/event-stream/event-server.ts

```diff
@@ -785,90 +785,112 @@ export async function startEventServer(opts: {
   );
 
   app.use(bodyParser.json({ type: 'application/json', limit: '500MB' }));
-  app.getAsync('/', (req, res) => {
+  app.get('/', (req, res) => {
     res
       .status(200)
       .json({ status: 'ready', msg: 'API event server listening for core-node POST messages' });
   });
 
-  app.postAsync('*', async (req, res) => {
-    const eventPath = req.path;
-    let payload = JSON.stringify(req.body);
-    await messageHandler.handleRawEventRequest(eventPath, payload, db);
-    if (logger.isDebugEnabled()) {
-      // Skip logging massive event payloads, this _should_ only exclude the genesis block payload which is ~80 MB.
-      if (payload.length > 10_000_000) {
-        payload = 'payload body too large for logging';
+  app.post(
+    '*',
+    asyncHandler(async (req, res, next) => {
+      const eventPath = req.path;
+      let payload = JSON.stringify(req.body);
+      await messageHandler.handleRawEventRequest(eventPath, payload, db);
+      if (logger.isDebugEnabled()) {
+        // Skip logging massive event payloads, this _should_ only exclude the genesis block payload which is ~80 MB.
+        if (payload.length > 10_000_000) {
+          payload = 'payload body too large for logging';
+        }
+        logger.debug(`[stacks-node event] ${eventPath} ${payload}`);
       }
-      logger.debug(`[stacks-node event] ${eventPath} ${payload}`);
-    }
-  });
+      next();
```

Comment:
This route did have a minor code change to add this `next()` call. The `@awaitjs/express` lib had a bug that caused the routes handlers below to be incorrectly called, synchronously, at the same time as this one.